recipes/cyvcf2/meta.yaml

```
@@ -35,7 +36,8 @@ requirements:
     - zlib
     - htslib >=1.10
     - openssl
-    - numpy
+    - numpy >=2.0.0  # [py >= 39]
+    - numpy >=1.16.0,<2.0.0  # [py < 39]
```

```suggestion
    - numpy  # [py < 39]
```
Any compatible version of numpy is fine and numpy 2.0 cannot be installed with python 3.8 anyway